src/main/resources/assets/css/main.less

```diff
@@ -570,6 +570,43 @@ fieldset[disabled] .btn-info.active {
   background-color: #8e929b;
 }
 
+.health-bar-column {
+  vertical-align: middle;
+
+  .health-bar {
```

Comment:
Let's hold off on nesting. And code in a classical css way. LESS was mainly introduced for the option of working with variables.
Maybe we can use variables here, so the colours become consistent?. Here are the green colours. Feel free do replace the other colours as well:
https://github.com/mesosphere/marathon/pull/914/files#diff-624b7a82640969c9d50881427621dc16R566
https://github.com/mesosphere/marathon/pull/914/files#diff-624b7a82640969c9d50881427621dc16L482
https://github.com/mesosphere/marathon/pull/914/files#diff-624b7a82640969c9d50881427621dc16L431

Or, at least we need to make them consistent.